src/server/vnparse.c

```diff
@@ -2486,6 +2481,139 @@ typedef struct resc_limit_entry {
 	resc_limit_t	*resc;
 } rl_entry;
 
+/* Reusing ATR_VFLAG_HOOK to mean that the resource key value pair was found
+ * in in execvnode
+ */
+#define IN_EXECVNODE_FLAG	ATR_VFLAG_HOOK
+
+#if !(defined(PBS_MOM) || defined(PBS_PYTHON))
+/**
+ * @brief
+ * compare lexicographically the names of resources in the resource list contained in
+ * two resc_limit_t structures
+ *
+ * @param[in]	left - the left resource limit.
+ * @param[in]	right - the right resource limit.
+ *
+ * @return int
+ * @retval -1	- left < right
+ * @retval 0	- left = right
+ * @retval 1	- left > right
+ */
+static int
+resc_limit_list_cmp_name(resc_limit_t *left, resc_limit_t *right)
```

Comment:
Check to make sure params 'left' and 'right' are non-NULL before referencing them.

Reply:
Still, please do the check.

Reply:
Done, see commit https://github.com/PBSPro/pbspro/pull/1452/commits/6151018baa0b74a7f013bf2e272e4ce090374365